cmd/mistryd/mistryd_test.go

```diff
@@ -246,11 +246,18 @@ func parseClientJSON(s string) (*types.BuildInfo, error) {
 // MISTRY_CLIENT_PATH environment variable or, if empty,  from the current
 // working directory where the tests are ran from.
 func cliBuildJob(args ...string) (string, string, error) {
+	return cliBuildJobTarget(target, args...)
+}
+
+func cliBuildJobTarget(trgt string, args ...string) (string, string, error) {
 	clientPath := os.Getenv("MISTRY_CLIENT_PATH")
 	if clientPath == "" {
-		clientPath = "./mistry"
+		clientPath = "/home/dtheodor/go/src/github.com/skroutz/mistry/mistry"
```

Comment:
This code should stay independent of the dev path.

Reply:
oops